db/migrate/20230124144937_create_likes.rb

```diff
@@ -0,0 +1,8 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+
+      t.timestamps
```

Comment:
- [ ] Kindly create references here as foreign keys between users table and likes table as `author_id`.
- [ ] Kindly create references here as foreign keys between posts table and likes table.